mobile/library/cc/engine_builder.h

```diff
@@ -65,7 +65,9 @@ class EngineBuilder {
   EngineBuilder& enforceTrustChainVerification(bool trust_chain_verification_on);
   EngineBuilder& enablePlatformCertificatesValidation(bool platform_certificates_validation_on);
   // Adds an RTDS layer to default config. Requires that ADS be configured
-  EngineBuilder& addRtdsLayer(const std::string& layer_name, int timeout_seconds = 5);
+  EngineBuilder& addRtdsLayer(const std::string& layer_name, const int timeout_seconds = 5);
+  // Adds a CDS layer to default config. Requires that ADS be configured
```

Comment:
super nit: add a period at the end of "configured", same for the RTDS comment above